django/contrib/postgres/fields/ranges.py

```diff
@@ -100,15 +101,21 @@ def db_type(self, connection):
         return 'int8range'
 
 
-class FloatRangeField(RangeField):
+class DecimalRangeField(RangeField):
     base_field = models.FloatField
     range_type = NumericRange
-    form_field = forms.FloatRangeField
+    form_field = forms.DecimalRangeField
 
     def db_type(self, connection):
         return 'numrange'
 
 
+class FloatRangeField(DecimalRangeField):
+    def __init__(self, *args, **kwargs):
+        warnings.warn('FloatRangeField has been renamed to DecimalRangeField and is now deprecated')
```

Comment:
`stacklevel=2` to have the warning show at `FloatRangeField` initialization time?